docs/reference/api/pages.md

```diff
@@ -220,7 +220,7 @@ Individual page objects will include `_children` and `_ancestor` arrays, as well
 
 ```javascript
 // Request inside an async function.
-const response = await fetch('http://example.net/api/v1/@apostrophecms/page/ckitdo5oq004pu69kr6oxo6fr:en:published?apikey=myapikey', {
+const response = await fetch('http://example.net/api/v1/@apostrophecms/page/ckitdo5oq004pu69kr6oxo6fr?apikey=myapikey&aposMode=published&aposLocale=en&renderAreas=true', {
```

Comment:
This is a great example too, but a separate one. The more common case is "I've got an _id, probably because I did a get all operation, and now I want a doc back with all the trimmings, and I don't want to think about modes and locales when that stuff is all in the _id already."

Reply:
It's true that you can send `aposDocId` instead of `_id` to leave off the current locale and mode of the document you already have, and specify the rest via query params (which always win if the info is present in the id), but that's not obvious nor is it being explained here.

Reply:
I added a sentence about the parameters overriding, but I can remove it.
I just adapted the existing examples to use query params instead of the `_id` to illustrate the usage.